### PR TITLE
misc: disable recipe.yaml for migrators

### PR DESCRIPTION
**Pull request by nichmor**
Description:

This PR prevents situations where new feedstocks would somehow trip up migrations. 

Checklist:

- [ ] Pydantic model updated or no update needed

Cross-refs, links to issues, etc:

### CODE REVIEW

File: conda_forge_tick/utils.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import json
 import logging
 import os
+from pathlib import Path
 import pprint
```
Comment (wolfv) [inline conda_forge_tick/utils.py:10]: this is a leftover?

Comment (nichmor) [reply conda_forge_tick/utils.py]: yes, thanks

**Comment by beckermr** [comment]
Can you fill out the PR description to provide the motivation for this one? I can think of easier ways to do this and am curious what the global plan is.

**Comment by wolfv** [comment]
@beckermr - we wanted to prevent a situation where new feedstocks would somehow trip up migrations. There are a couple places where the `meta.yaml` file is unconditionally loaded. 

Any hints / ideas welcome!

**Comment by beckermr** [comment]
Thanks @wolfv! That makes sense. Let me think on this a bit.

**Comment by wolfv** [comment]
@beckermr any new ideas? :) Sorry for the turmoil

**Review by beckermr — CHANGES_REQUESTED** [review]
No worries on the turmoil! Thanks for the hard work. So I had a few ideas but I ended up hating all of them. Here is what we should do.

1. This PR has missed some of the migrators (the graph / ABI ones). We should fix that unless it was purposeful in which case we should discuss.

2. Let's make a single tracking issue that has a list of the skipped migrators to figure out if we will enable for v2 and if we do, we can link to the PR.

**Comment by nichmor** [comment]
> No worries on the turmoil! Thanks for the hard work. So I had a few ideas but I ended up hating all of them. Here is what we should do.
> 
> 1. This PR has missed some of the migrators (the graph / ABI ones). We should fix that unless it was purposeful in which case we should discuss.
> 2. Let's make a single tracking issue that has a list of the skipped migrators to figure out if we will enable for v2 and if we do, we can link to the PR.

Hey @beckermr ! sorry if I'm being slow - could you please point me where it is the graph or abi migrator located?

**Comment by beckermr** [comment]
It is in the file called `core.py` in the same module. There appear to a lot of other migrators we have missed. Are those OK? We need to check all of them.

**Comment by wolfv** [comment]
@beckermr - I searched but didn't find anything in `core.py`.

**Review by beckermr — CHANGES_REQUESTED** [review]
So I don't get why this PR doesn't just add statements in the filter functions to bypass v1 recipes.

**Pull request closed without merging**